policy/policy.example.agent.json

```diff
@@ -0,0 +1,35 @@
+{
+    "description": "hello there, security policies are fun!",
+    "rules": [
+      {
+        "is_stateful": true,
+        "ports": [
+          80,
+          443
+        ],
+        "protocol": "TCP"
+      }
+    ],
+    "peers": [
+      {
+        "segment_network_id": 2
+      }
+    ],
+    "direction": "ingress",
+    "applied_to": [
+      {
+        "segment_network_id": 1,
+        "tenant_network_id": 1
+      }
+    ],
+    "datacenter" : {
```

Comment:
Using config files brings same exact problems - you no longer have a single source of truth and have to maintain similar config in different places.
A better solution would be to do a discovery via romana, and we will implement when we will be porting the thing to Golang.